boards/riscv/longan_nano/longan_nano_defconfig

```diff
@@ -0,0 +1,28 @@
+# Sipeed longan nano Development Board Configuration
+#
+# Copyright (c) 2021 Tokita, Hiroshi <tokita.hiroshi@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_SOC_SERIES_RISCV_GIGADEVICE_GD32VF103=y
+CONFIG_SOC_GD32VF103CBT6=y
+
+# 108MHz for system clock
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=108000000
+
+# 27MHz for machine timer
+CONFIG_RISCV_MACHINE_TIMER=y
+CONFIG_RISCV_MACHINE_TIMER_SYSTEM_CLOCK_DIVIDER=2
+
+CONFIG_RISCV_SOC_MCAUSE_EXCEPTION_MASK=0x7FF
+
+CONFIG_BUILD_OUTPUT_HEX=y
```

Comment:
this needs to be activated ad SoC level, please check https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/arm/gigadevice/Kconfig#L7

Reply:
I got it.
Moving to soc/riscv/riscv-privilege/nuclei/gd32vf103/Kconfig.series.